### ISSUE TITLE
Multiple loading of plugins and skin.url

### ISSUE THREAD

**Issue by artur-michalak** [jwplayer/jwplayer#2981]
### Expected Behavior
Plugins and skin should load only once.

### Actual Behavior
When new player is created then the same JW Player related scripts are added to HEAD.

![image](https://user-images.githubusercontent.com/2443904/41971488-e328801a-7a0e-11e8-969a-961098743d52.png)
### Steps to reproduce

1. Create html
2. Add this code
```javascript
    var player = null,
        count = 0,
        intervalId;

    function createPlayer() {
        player = jwplayer("player1").setup({
            "playlist": [{
                "file": "http://content.jwplatform.com/videos/bkaovAYt-640.mp4"
            }],
            skin: {
                url: '/css/player.css',
                name: 'svp'
            },
            autostart: false,
            "key": "KEY",
        });
    }

    intervalId = setInterval(function () {
        count++;
        // cleanup previous
        if(player) {
            player.remove();
        }

        createPlayer();

        if (count === 4) {
            clearInterval(intervalId);
            player.remove();
        }
    }, 500);
```
3.

### Environment

- *hardware | device* PC
- *operating system | version* Windows 10
- *browser | version* Chrome 67
- *JW Player version | url* 8.4.0

----------------------------------------------------------------------------------------

This template is provided as a guide to filing bugs and feature requests. Please replace 
all text *in italics* with details describing your issue. When submitting a feature request, 
"Steps to reproduce" can be substituted with a user story or list of acceptance criteria.

**Comment by robwalch** [comment]
Hi @artur-michalak,

Thank you for the bug report. Is this happening only when removing a player before setting up a new one (If you call setup on an existing player instance, the current instance will be replaced/removed for you)?

**Comment by artur-michalak** [comment]
It has the same bug when setup is called on player instance, however it works well when .load() is used

**Comment by robwalch** [comment]
Hi @artur-michalak,

I'm going to make sure we only load plugin urls's once on a page, but for CSS, I think it's on you to change your setup. skin loading is like a courtesy. Ideally you should embed it on the page yourself for best performance. If you are setting up multiple players, specify the url on the first player, and then only include "name" on subsequent setups. That would ensure that you only load the CSS once. The player can't know that that your CSS URL returns static content and should only load it once.

**Comment by robwalch** [comment]
Hi @artur-michalak,

We have a note here that using `skin.url` is not preferred https://developer.jwplayer.com/jw-player/docs/developer-guide/customization/css-skinning/skins_creating/#option-2-configuring-the-css-url-within-skinurl which is linked to from https://developer.jwplayer.com/jw-player/docs/developer-guide/customization/configuration-reference/#custom-skins (I would agree that details in the configuration-reference would be useful).

Could you could provide a specific example of the doc you used and what you would expect to see?

**Comment by robwalch** [comment]
@artur-michalak The plugin loading fix is available in v8.4.2.
